.github/workflows/config/update_finetune_config_on_intel_gpu.py

```diff
@@ -3,7 +3,7 @@
 
 
 def update_finetune_config(base_model):
-    conf_file = "finetune/finetune.yaml"
+    conf_file = "llmonray/finetune/finetune.yaml"
```

Comment:
`llmonray` is cluttered and hard to read. use `llm_on_ray` instead just like intel_extension_for_pytorch